client/src/main/java/uk/co/blackpepper/bowman/Client.java

```diff
@@ -118,16 +121,28 @@ public URI post(T object) {
 		
 		return resourceUri;
 	}
-
+	
+	/**
+	 * PUT the given entity to the entity's collection resource. This will not update associations!
```

Comment:
The last part of this comment ('this will not update associations') isn't necessarily true - as currently implemented, the associations *are* sent. We only know this to be true when using a Spring Data REST back end, and even then only under certain circumstances. Please omit this part of the comment, as I don't think we should be defining the behaviour of the back end in the client.